Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Internal_-_Generic_v2.yml

```diff
@@ -428,17 +489,17 @@ inputs:
   description: The IP address to enrich.
   playbookInputQuery:
 - key: InternalRange
-  value:
-    simple: inputs.InternalRange
+  value: {}
```

Comment:
should we add the default internal range according to the RFC 1918?

Reply:
Ill add it back